changelog.md

```diff
@@ -5,6 +5,7 @@
 ### Changes
 
 - [#3959](https://github.com/ignite/cli/pull/3959) Remove app name prefix from the `.gitignore` file
+- [#3962](https://github.com/ignite/cli/pull/3962) Rename `map`/`list`/`single` query type names and cli commands
```

Comment:
I think the better description that @julienrbrt mentioned meant to be added to the changelog entry

Reply:
Oops! Sorry, I forgot to push my changes 😅